includes/import-products.php

```diff
@@ -18,11 +18,26 @@
  * @throws Exception
  */
 function dw_import_products() {
+	global $dw_options;
+
 	$client  = RestApiClientFactory::getClient();
 	$courses = $client->getCourseInstances(new DateTime(), null) ?? [];
-
 	$createdProducts = [];
 
+	if( $dw_options['ccvCode'] && $dw_options['ccvCode'] !== "") {
+		$codesToFilter = explode(';', $dw_options['ccvCode']);
```

Comment:
Maybe move this to a separate function? Also, I can't find tests for the filtering.